### PR TITLE
added section headings to FAQ

### PR DESCRIPTION
**Pull request by nerik**
screen shot 2016-08-22 at 11 35 39

### CODE REVIEW

File: app/styles/_base.scss [hunk 1]
```diff
@@ -81,3 +81,14 @@ a {
...
+.section-title {
+  color: $color-9;
+  display: block;
+  font-size: $font-size-xx-big;
+  font-weight: $font-weight-normal;
+  line-height: 1.05;
+  margin: 60px 0 18px;
+  max-width: 700px;
+  width: 100%;
+}
```
Comment (nerik) [inline app/styles/_base.scss]: This was almost duplicated from blog's c-list-posts.scss. It should probably be a base style, not at component level.

**Comment by andresgnlez** [comment]
@nerik we are using global styles for links (check  `_base.scss`). It should be valid for your link adding the class `-underline`.

File: app/styles/_base.scss (hunk 1 above)
Comment (andresgnlez) [inline app/styles/_base.scss]: Agree. We should do something with all those common styles...

**Comment by andresgnlez** [comment]
Ignore last comment. Wrong PR.

**Pull request merged**